include/ppx/metrics.h

```diff
@@ -167,12 +175,12 @@ class MetricGauge final : public Metric
     }
     METRICS_NO_COPY(MetricGauge)
 
-    Stats ComputeStats() const;
+    GaugeComplexStatistics ComputeComplexStats() const;
 
 private:
     MetricMetadata               mMetadata;
     std::vector<TimeSeriesEntry> mTimeSeries;
-    Stats                        mBasicStats;
+    GaugeBasicStatistics         mBasic;
```

Comment:
nit: can we keep the name as `mBasicStats`? Otherwise, it is not clear "basic" what?

Reply:
fixed